tox.ini

```diff
@@ -1,27 +1,46 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy,pypy3,docs
+    py27,
+    py33,
+    py34,
+    py35,
+    py36,
+    pypy,
+    pypy3,
+    docs,
+    coverage-report,
 
 [testenv]
+usedevelop = true
```

Comment:
Without this `coverage` has to use the paths where the egg is installed: `.tox/py27/...`, …. Thus it is not able to combine the coverage for the different Python versions.